cumulusci/cumulusci.yml

```diff
@@ -256,6 +256,10 @@ tasks:
         description: Retrieves a list of the names of any permission sets assigned to the running user.
         class_path: cumulusci.tasks.preflight.permsets.GetPermissionSetAssignments
         group: Salesforce Preflight Checks
+    get_available_permission_set_licenses:
```

Comment:
@vmarasow  This should be get_available_permission_sets, right?

Also could you please adjust this to NOT rewrite most of the whitespace in cumulusci.yml? (see diff below)

Reply:
Good catch. I got all the whitespace back to the original state.